pubspec.yaml

```diff
@@ -62,6 +62,7 @@ dependencies:
   google_maps_flutter: ^2.1.1
   geolocator: ^8.2.0
   geocoding: ^2.0.2
+  google_maps_webservice: ^0.0.20-nullsafety.5
```

Comment:
why is this needed?

Reply:
Dont like this library, last update was 10 moth ago and it has 60 poits out of 130

Reply:
It's the package used for Google search places

Reply:
> Dont like this library, last update was 10 moth ago and it has 60 poits out of 130

I see ok I'll take a look on others pancakes or even make our own implementation it's just a https google calls